src/ensembl/src/shared/components/layout/components/sidebar-modal/SidebarModal.test.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import faker from 'faker';
+
+import SidebarModal, { SidebarModalProps } from './SidebarModal';
+
+const sidebarModalContent = 'This content goes into the sidebarModal’s content';
+
+const sidebarModalTitle = 'This content goes into the sidebarModal’s title';
+
+const onClose = jest.fn();
+
+const classNames = {
+  wrapper: faker.lorem.word(),
+  title: faker.lorem.word(),
+  content: faker.lorem.word(),
+  closeButton: faker.lorem.word()
+};
```

Comment:
Do we really need all these classes? Doesn't `SidebarModal` define positions of the elements that should remain the same across all sidebar modals throughout the site?

Reply:
Removed these classes